pandas/tests/resample/test_time_grouper.py

```diff
@@ -333,26 +333,97 @@ def test_upsample_sum(method, method_args, expected_values):
     tm.assert_series_equal(result, expected)
 
 
-def test_groupby_resample_interpolate():
+@pytest.fixture
+def groupy_test_df():
+    df = DataFrame({"price": [10, 11, 9], "volume": [50, 60, 50]})
+    df["week_starting"] = date_range("01/01/2018", periods=3, freq="W")
+    return df.set_index("week_starting")
```

Comment:
```suggestion
    return DataFrame({"price": [10, 11, 9], "volume": [50, 60, 50]}, index=date_range("01/01/2018", periods=3, freq="W"))
```

Will probably also need formatting by pre-commit